compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/SkikoUIView.uikit.kt

```diff
@@ -34,9 +30,13 @@ import platform.Metal.MTLPixelFormatBGRA8Unorm
 import platform.QuartzCore.CAMetalLayer
 import platform.UIKit.*
 
+internal interface RenderDelegate {
```

Comment:
It seems very similar to existing `SkikoView` interface. Can we align it with the existing one? Maybe just naming

Reply:
This interface comes from SkikoUIViewDelegate. I think better to stay function names as is for now.

Reply:
`SkikoView` interface exists for the sole purpose of abstracting `skiko` from `core`, it was removed for iOS because it basically did nothing and collapsed during moving `skiko` window management for iOS to `core`.

The delegate, on the other side, reflect very specific aspects of `SkikoUIView` functionality that it, well, _delegates_ somewhere else.

The fact that it was separated into two separate ones to reflect that two different entities now implement parts of the former interface in my opinion could be treated as violating `dependency inversion principle`:
> Abstractions should not depend on details.

But since those two details are indeed a bit _unrelated_ parts of functionality, I think the resulting solution is fine.